### ISSUE TITLE
Possibility of other storage types?

### ISSUE THREAD

**Issue by verdurin** [ComputeCanada/magic_castle#36]
Hello

From a first look it appears that `NFS` is the current default method for providing shared storage to the cluster.

I am looking to provision clusters on public cloud for bioinformatic applications that are particularly heavy on I/O. Hence I would prefer to be able to provision something like Lustre and consume that in the cluster. 

Is that sort of flexibility something you are considering or would consider?

**Comment by cmd-ntrf** [comment]
Yes, we are planning to add other types of storage and Lustre is the first on our list. @guilbaults already did an implementation with earlier version of Magic Castle.

Once we are done, it should be as easy as replacing "nfs" by "lustre" in storage type of your main.tf.

I am however unable to provide an exact timeline, but you should expect some commits related to this in Q2 and it should be part of a release in Q3.
